sapi/fpm/tests/bug75712-getenv-server-vars_001.phpt

```diff
@@ -0,0 +1,68 @@
+--TEST--
```

Comment:
NIT: I usually prefer not to use 001 names in FPM to see immediately from the test name what it is for which was quite useful for me. But it's not a huge issue if there are few such tests.